extensions/partitioning.md

```diff
@@ -35,8 +35,8 @@ those events.
 The partitionkey attribute extension uses the key `partitionkey` for
 in-memory formats.
 
-### Transport format
+### Protocol format
```

Comment:
I'm not sure if *in-memory format* vs. *protocol format* are still the right categories. What about *stand-alone event format* and *protocol binding*? Or *structured-mode message* vs. *binary-mode message*?

Reply:
This feels like more than a syntax fix so I'd prefer to leave that for a follow-on PR - if you want to open that one up